sunpy/map/compositemap.py

```diff
@@ -433,6 +433,10 @@ def plot(self, axes=None, annotate=True,  # pylint: disable=W0613
             # non False levels property.  If levels is False, then the layer is
             # rendered using imshow.
             if m.levels is False:
+                # Check if the linewidths argument is provided, if so, then delete it from params.
+                if matplot_args.get('linewidths', -1) != -1:
```

Comment:
Why `-1`? wouldn't `get` will provide `None` by default? This will work:
```python
if matplot_args.get('linewidths'):
```
Unless there's a possibility that the user has `linewidths = 0`, but... even then, what would htat achieve?

Reply:
Actually, it was just habitual, yeah `if matplot_args.get('linewidths')` would also work, logically it's the same. I don't think this implementation would cause any problem. If you think that this implementation is wrong, I can change it.

Reply:
In favour of readability then we should simplify it to what I suggested.

Reply:
Sure, will do it !